packages/grid/_modules_/grid/components/SelectedRowCount.tsx

```diff
@@ -1,15 +1,21 @@
 import * as React from 'react';
+import { ApiContext } from './api-context';
 
 interface SelectedRowCountProps {
   selectedRowCount: number;
 }
 
 export function SelectedRowCount(props: SelectedRowCountProps) {
   const { selectedRowCount } = props;
+  const apiRef = React.useContext(ApiContext);
+  const rowSelectedText =
+    selectedRowCount > 1
+      ? apiRef!.current.getLocaleText('footerRowSelectedPlural')
+      : apiRef!.current.getLocaleText('footerRowSelected');
```

Comment:
The plural isn't universally defined by `> 1`. For instance, it doesn't work in the default locale, in en_US it's `!== 1`: 0 cats, 1 cat, 2 cats. https://www.quora.com/Why-is-0-plural 🤷‍♂️

How about we expose a function here, so we allow developers to implement what they need?

```suggestion
  const rowSelectedText = apiRef!.current.getLocaleText('footerRowSelected')(selectedRowCount);
```